models/models.js

```diff
@@ -18,14 +18,15 @@ exports.retrieveTopics = () => {
 };
 
 exports.retrieveArticleById = (article_id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    return db.query(`SELECT a.*, COUNT(c.body) AS comment_count FROM articles a LEFT OUTER JOIN comments c ON c.article_id = a.article_id WHERE a.article_id = $1 GROUP BY a.article_id;`, [article_id])
```

Comment:
Great work tweaking this query to return a comment count.

One thing to consider - this currently sends the comments back as a string. This is manageable, because we can convert to a number on our frontend later on, but we'd save ourselves the trouble if we get a number out on this end.

Look into using either CAST with AS INTEGER or look into adding ::INT as a way to change the type.